ruby3.2-jrjackson.yaml

```diff
@@ -33,32 +23,14 @@ pipeline:
       repository: https://github.com/guyboertje/jrjackson
       tag: v${{package.version}}
 
-  - uses: maven/configure-mirror
-
-  - runs: |
-      gem install bundler
-      gem install ruby-maven
-      gem install maven-tools
-      gem install jar-dependencies
-
-  - runs: sed -i "s/3\.1/3.12.1/g; s/1\.8/11/g" Mavenfile
-
-  - runs: mvn clean install -Dmaven.test.skip
+  - uses: patch
+    with:
+      patches: mavenfile.patch
 
   - runs: |
-      # Extract the jackson_version to use in the maven dependency:copy commands
-      export JACKSON_VERSION=$(grep -A 1 'def self.jackson_version' ./lib/jrjackson/build_info.rb | tail -n 1 | awk -F "'" '{print $2}')
-
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-annotations/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-core/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/core/jackson-databind/${JACKSON_VERSION}
-      mkdir -p ./lib/com/fasterxml/jackson/module/jackson-module-afterburner/${JACKSON_VERSION}
-      mkdir -p ./lib/jrjackson/jars
-
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-annotations:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-annotations/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-core:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-core/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.core:jackson-databind:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/core/jackson-databind/${JACKSON_VERSION}
-      mvn dependency:copy -Dartifact=com.fasterxml.jackson.module:jackson-module-afterburner:${JACKSON_VERSION} -DoutputDirectory=lib/com/fasterxml/jackson/module/jackson-module-afterburner/${JACKSON_VERSION}
+      bundle install
+      rake
```

Comment:
Just a sanity check, I saw a `vendor_jars` task in the Rakefile. Does that vendor happen in the `lib` directory so it gets packaged and installed with the gem or does it drop it in a different directory?

Reply:
The JAR files are placed in the `vendor` directory at the end but I'm not entirely sure about the mechanism. I found this in the jrjackson build files: https://github.com/guyboertje/jrjackson/blob/86e21566467349cb92ee5687f2933335ee227c93/lib/jrjackson/build_info.rb#L37-L45

Reply:
Do they get bundled in the gem itself so they come with the APK?

Reply:
Looks like it does!